development/webpack/webpack.config.ts

```diff
@@ -260,6 +260,12 @@ const config = {
     rules: [
       // json
       { test: /\.json$/u, type: 'json' },
+      // snaps
```

Comment:
While this change is introduced to support snaps, it ends up enabling a lot more that than. Can you write a better comment akin to explaining that it treats `.json` paths  in `new URL(string, import.meta.url) as assets?

Reply:
applied.